react/react2/week1/search-app/git-app/src/components/SearchUser.js

```diff
@@ -0,0 +1,35 @@
+import React, { useEffect, useState } from 'react';
+import User from './User';
+
+function SearchUser({ username }) {
+  const [isLoading, setIsloading] = useState(true);
+  const [user, setuser] = useState([]);
+
+  useEffect(() => {
+    fetch(`https://api.github.com/search/users?q=${username}`)
+      .then((response) => response.json())
+      .then((data) => {
+        setIsloading(false);
+        setuser(data.items);
+      });
+  }, [username]);
+  return (
+    <div>
+      {typeof user === 'undefined' ? (
```

Comment:
you are fetching data, which type should be defined, I think here you are looking for an error if fetch fails